src/dropdown/abstract-dropdown-view.class.ts

```diff
@@ -60,13 +72,21 @@ export class AbstractDropdownView {
 	 * Returns the `HTMLElement` for the item that is selected within the `DropdownList`.
 	 */
 	getCurrentElement(): HTMLElement { return; }
+	/**
+	 * Returns the current items as an Array
+	 */
+	getListItems(): Array<ListItem> { return; }
```

Comment:
Is this so items wouldn't accidentally return Observable?
Right now getters always return array of ListItems

Reply:
It's two fold, and I should update the comment, but `getListItems` is guaranteed to always return an `Array<ListItem>` while the getter/setter (due to typescript constraints) can either return an Array or an Observable ... Really I suppose I should update the getter/setter to return the value passed, especially since we have this method now